### PR TITLE
I have uploaded all the required files in this.

### LINKED ISSUE #42: Stop_watch Page
**Project title**
Counter

**Project Description**
This is a stopwatch counter project made using basic tech stack,.
@khushi-purwar If you find this fit, please assign me this  issue.

**Stack**:  
t.

- [x] Html
- [x] CSS
- [x] JavaScript

### PR DESCRIPTION
**Pull request by deepu0201**
<hr>

## Description 📜

I have added a project of mine, which is a counter page.

Fixes #42 

<hr>

## Type of change 📝

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] Add a new project

<hr>

## Domain of Contribution 📊

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] Project using only HTML & CSS
- [x] Project using HTML, CSS & JS

<hr>
 
## Checklist ✅

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Optional) 📸
Add the screenshot of the changes you have made

output1

### CODE REVIEW

**Comment by deepu0201** [comment]
@meinhoonharsh please review and let me know if anything is needed

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
Started looking at HTML file, found some changes 🙃

File: Counter/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        
+        <title> Uhini's Counter</title>
```
[the remark this answers is not in the document]
Comment (uhini0201) [reply Counter/index.html:5]: Okk will change that @meinhoonharsh

**Comment by deepu0201** [comment]
@meinhoonharsh updated it.

**Review by khushi-purwar — APPROVED** [review]

**Pull request merged**